src/NuGet.Services.AzureSearch/SearchService/ParsedQuery.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.AzureSearch.SearchService
+{
+    public class ParsedQuery
+    {
+        public ParsedQuery(string text, string packageId)
+        {
+            Text = text;
+            PackageId = packageId;
+        }
+
+        public string Text { get; }
```

Comment:
I would add a comment. Considering the type name, one could assume this is the inputted NuGet query, not the outputted Azure Search query.

Reply:
Actually "query" is how we refer to the user input and the Azure Search SDK accepts "text". But a comment sounds good.